i18n/en.json

```diff
@@ -58,7 +58,7 @@
   "add_command.method.help": "The type of command request issued to the Request URL.",
   "add_command.method.post": "POST",
   "add_command.save": "Save",
-  "add_command.token": "<b>Token</b>: {token}",
+  "add_command.token": "**Token**: {token}",
```

Comment:
Only changes should be made in `en.json`.  Please revert changes to other languages.  Other languages will be handled separately by our translation server/team.